docs/services/navigation/_index.md

```diff
@@ -13,7 +13,7 @@ tags: ["navigation", "services", "base", "rover"]
 
 The Navigation service is the stateful definition of Viam's [Motion Service](/services/motion/).
 It uses GPS to autonomously navigate a rover [base](/components/base/) to user defined endpoints called `Waypoints`.
-Once the `Waypoints` are added and the `Mode` of the service is [set to `waypoints`](#setmode), the service communicates with the base as it begins to navigate to the `Waypoints`.
+Once these waypoints are added and the mode of the service is [set to `waypoints`](#setmode), the service begins to define the robot's path.
 
 ## Configuration
```

Comment:
@npentrel want to clear up here-- Do you have any suggestions for how we can make the formatting here look prettier for the "store" attribute? From what I can tell there is no default value. I tried looking around but don't have a better description here atm, don't think Raymond did either

Reply:
I'd condense it to this:
```
| `store` | obj | **Required** | The configuration object to use for the Navigation data storage. Either `[{"type": "memory"}]` or `[{ "type": "mongodb", "config": {"uri": "mongodb://127.0.0.1:27017"} }]`. |
```

We do still need to add more information as to what the memory store vs mongodb store is and how a user should choose - did you get any more info for that?

Reply:
@raybjork I don't think I got clarity on this when we discussed this attribute-- do you know how the user should choose type of store? Or know of anyone who might have worked on this before that might know, like Rand?

Reply:
so memory is volatile and the nav points last as long as the process is running. using mognodb will persist them to whatever URI and will persist through process restarts

Reply:
@edaniels Thanks!